testsuite/features/support/constants.rb

```diff
@@ -225,3 +228,75 @@
                       'ubuntu1804_minion' => 'x86_64',
                       'ubuntu2004_ssh_minion' => 'x86_64',
                       'ubuntu2004_minion' => 'x86_64' }.freeze
+
+CHANNEL_TO_SYNCH_BY_OS_VERSION = {
```

Comment:
Very nice, I love it! Thanks :)

Reply:
Haha You forgot it! It was **you** suggestion in a previous PR, so I brought here ;-)